ui/src/components/Header.tsx

```diff
@@ -6,28 +6,23 @@ import Button from "@mui/material/Button";
 import Toolbar from "@mui/material/Toolbar";
 import React, { FC } from "react";
 
-type HeaderProps = {
-  csrfToken: string;
-  signOutUrl: string;
-};
-const Header: FC<HeaderProps> = ({ csrfToken, signOutUrl }) => {
+type HeaderProps = {};
+const Header: FC<HeaderProps> = ({}) => {
+  function handleClickLogout(evt: React.MouseEvent) {
+    evt.preventDefault();
+    location.href = "/.pomerium/sign_out";
+  }
+
   return (
     <AppBar position="sticky">
       <Toolbar>
         <a href="/.pomerium">
           <Logo />
         </a>
         <Box flexGrow={1} />
-        {signOutUrl ? (
-          <form action={signOutUrl}>
-            <CsrfInput csrfToken={csrfToken} />
-            <Button variant="text" color="inherit" type="submit">
-              Logout
-            </Button>
-          </form>
-        ) : (
-          <></>
-        )}
+        <Button variant="text" color="inherit" onClick={handleClickLogout}>
```

Comment:
This page was completely changed in my other PR but I can merge this in and resolve.